walsamlee/politico
# #163416863 Add Create Political Office Endpoint

#### What does this PR do?
Add `create-office` API endpoint
#### Description of Task to be completed?
Have `/api/v1/offices` endpoint working
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ft-v1-create-offices-163416863`
- Using postman, `POST` with API endpoint URL `/api/v1/offices` 
#### What are the relevant pivotal tracker stories?
#163416863
